src/assets/scripts/client/game/GameController.js

```diff
@@ -362,12 +361,12 @@ class GameController {
      * @method update_pre
      */
     update_pre() {
+        this.game.delta = Math.min(TimeKeeper.deltaTime * this.game.speedup, 100);
```

Comment:
this line concerns me.  it appears as though we're keeping track of `deltaTime` in a few places.  This will need to be looked at in the near future